Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/NICOLE.MSG

```diff
@@ -188,8 +188,7 @@
 {284}{}{Can I ask a few more questions?}
 {285}{}{That was all for now. I gotta go.}
 
-# 286 changed with patch 1.3 from TeamX
-{286}{}{Please, we're busy. Don't you have things to do?}
+{286}{Nic_73}{Please, we're busy. Don't you have things to do?}

```

Comment:
This voice file doesn't exist.

Reply:
I reverted it to vanilla with the float_msg in NICOLE.SSL replaced w/ **NMessage()**. I looked at MORPH.SSL and saw that he speaks message 116 with BMessage instead of a float.

Reply:
I took a look at the first vanilla files again, and it seems to be using Nic_72 and not _73.

Reply:
Glad that it was correct in the end!